apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPool.DatabaseStatistics.cs

```diff
@@ -20,14 +20,25 @@ namespace Google.Cloud.Spanner.V1
     public partial class SessionPool
     {
         /// <summary>
-        /// A snapshot of statistics for one database within a <see cref="SessionPool"/>.
+        /// A snapshot of statistics for a SessionPoolKey within a <see cref="SessionPool"/>.
         /// </summary>
         public sealed class DatabaseStatistics
```

Comment:
And ideally we also deprecate this class and create one exactly like this one but named `SessionPoolKeyStatistics`.